test/webpack.build.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs-extra');
```

Comment:
Still `import/order no-param-reassign` are 'conflicting' with the loader logic, the first for style/readability the latter for sourcemaps

```js
import fs from 'fs'
import path from 'path'

import { getOptions } from 'loader-utils'
import { validateOptions } from 'schema-utils'

import helper from './helper'
```

```js
if (options.sourceMap) {
  map = updateMap(map)
}
```